install/helm/vault-sync/templates/secret.yaml

```
@@ -6,4 +7,7 @@ metadata:
     {{- include "vault-sync.labels" . | nindent 4 }}
 type: Opaque
 data:
-  {{- toYaml .Values.secrets | nindent 2 }}
+{{- range $key, $entry := .Values.secrets }}
+  {{ $key }}: {{ $entry | quote }}
+  {{- end }}
+{{- end }}
```

nit: trailing end of line
```suggestion
{{- end }}

```

---

Not sure how my linter missed that sorry.